app/forms/work_packages/types/subject_configuration_form.rb

```diff
@@ -31,60 +31,55 @@
 module WorkPackages
   module Types
     class SubjectConfigurationForm < ApplicationForm
-      class << self
-        def has_pattern?(type)
-          type.replacement_pattern_defined_for?(:subject)
-        end
-      end
-
       form do |subject_form|
         subject_form.radio_button_group(name: :subject_configuration) do |group|
           group.radio_button(
-            value: "manual",
-            checked: !has_pattern?,
+            value: :manual,
+            checked: subject_configuration_manual?,
             label: I18n.t("types.edit.subject_configuration.manually_editable_subjects.label"),
             caption: I18n.t("types.edit.subject_configuration.manually_editable_subjects.caption"),
             data: { action: "admin--subject-configuration#hidePatternInput" }
           )
           group.radio_button(
-            value: "auto",
-            checked: has_pattern?,
+            value: :generated,
+            checked: !subject_configuration_manual?,
             label: I18n.t("types.edit.subject_configuration.automatically_generated_subjects.label"),
             caption: I18n.t("types.edit.subject_configuration.automatically_generated_subjects.caption"),
-            disabled: !EnterpriseToken.active? && !has_pattern?,
+            disabled: !enterprise? && subject_configuration_manual?,
             data: { action: "admin--subject-configuration#showPatternInput" }
           )
         end
 
         subject_form.group(data: { "admin--subject-configuration-target": "patternInput" }) do |toggleable_group|
           toggleable_group.text_field(
-            name: :subject_pattern,
-            value: subject_pattern,
+            name: :pattern,
             label: I18n.t("types.edit.subject_configuration.pattern.label"),
             caption: I18n.t("types.edit.subject_configuration.pattern.caption"),
             required: true,
-            input_width: :large
+            input_width: :large,
+            validation_message: validation_message_for(:patterns)
           )
         end
 
         subject_form.submit(
           name: :submit,
           label: I18n.t(:button_save),
-          disabled: !EnterpriseToken.active? && !has_pattern?,
```

Comment:
This was part of a fix that I specifically performed after QA noticed that the submit button remained enabled, even if the rest of the form is disabled.

I agreed that it's confusing to be able to submit a form that allows no changes to be made.

Reply:
I talked it through with dominic just today. The application of the form model makes the disabling of the submit button obsolete. I can explain, if need be :D

Reply:
As discussed IRL: I can kinda live with most outcomes, but it would make sense to align our wireframes to match what we want to see, so that QA also knows what to expect.

Reply:
Yes, I will talk to Dominic and cecile.

Reply:
As discussed IRL: We leave submit button active and I update the AC.